habitat_sim/agent/controls/default_controls.py

```diff
@@ -18,67 +25,79 @@
 _Z_AXIS = 2
 
 _rotate_local_fns = [
-    hsim.SceneNode.rotate_x_local,
-    hsim.SceneNode.rotate_y_local,
-    hsim.SceneNode.rotate_z_local,
+    SceneNode.rotate_x_local,
```

Comment:
The refactoring is a large part of the patch which makes it harder to review. As I'm scrolling up and down, I find myself working extra hard to ignore the refactoring bits so that I can focus on the change.

Reply:
Yeah.. I really wish git/github did a better job at supporting things like this -- making another PR and then dealing with the resulting merge conflicts is painful, which is why I tend to do it this way.

Reply:
One idea would be to put the refactoring bits into a separate commit but part of the same PR.